app/libs/js/LEAF/formGrid.js

```diff
@@ -119,44 +230,26 @@ var LeafFormGrid = function (containerID, options) {
     var virtualHeader = '<tr id="' + prefixID + "tVirt_tr" + '">';
     if (showIndex) {
       temp +=
-        '<th id="' +
+        '<th tabindex="0" id="' +
         prefixID +
         'header_UID" style="text-align: center">UID</th>';
-      virtualHeader += '<th style="text-align: center">UID</th>';
+      virtualHeader +=
+        '<th id="Vheader_UID" style="text-align: center">UID</th>';
     }
     $("#" + prefixID + "thead").html(temp);
 
     if (showIndex) {
```

Comment:
Why are there two if-checks for showIndex?  One right after the other.  Couldn't they be combined?